test/members-test.js

```diff
@@ -0,0 +1,232 @@
+const chai = require('chai')
+const chaiHttp = require('chai-http');
+const path = require('path')
+const expect = chai.expect
+
+chai.use(chaiHttp);
+
+const imagen =  path.join(__dirname, '/imgTest/member-test.png');
+
+const URL = 'http://localhost:3000';
+let token;
+
+describe('Suite de peticiones GET a members',()=>{
+
+    it('Deberia generar el token del usuario',(done) => {
+        chai.request(URL)
+            .get('/auth/login ')
+           .set({email:"test@test.com",password:"1234"})
+            .end((error, res) => {
+               expect(res.body).to.have.property('token');
+                token = res.body.token;
```

Comment:
I think it will be better to use directly the `generateAccesToken` function because if this request fail somehow, all the following test that require the value of **token** will fail as well.

Reply:
Pero eso debería manejarse desde el servidor, aquí al hacer login automáticamente debería mandarme el token si no hay ningún error en el controlador de User